content/archives/old-tech-blog/entries/ccsearch-testing-progress/contents.lr

```diff
@@ -9,8 +9,8 @@ body:
 
 Two months ago we set out to develop a brand new interface for [search.creativecommons.org](http://search.creativecommons.org). During that time, two UIs have been in testing alongside the current search. All visitors have had the option to switch their experience over to one of the beta versions.
 
-  1. [A drop-down list](/demos/search/search1) of supported search engines
```

Comment:
please undo the changes to this file since this data was imported from an old blog and the content shouldn't be updated